core/src/main/java/org/fao/geonet/util/XslUtil.java

```diff
@@ -1300,6 +1300,13 @@ public static String md5Hex(String str) {
         return org.apache.commons.codec.digest.DigestUtils.md5Hex(str);
     }
 
+    public static String md5HexIgnoreWhiteSpaces(String str) {
```

Comment:
I don't think you do this.   This will probably create the error that we had at the beginning of this PR where XslUtil.java is not available from ISO19139 schema due to circular dependencies.

Why do you need to strip spaces and returns?  This is not user input and therefore we should have control of this value?  If you really need to cleanup spaces you may want to try normalize-space() in the xslt instead.

i.e. 
`util:md5Hex(string(exslt:node-set(normalize-space(.))))`

Reply:
ok, i tried this approach, and it works. I should also fix the unit test failure. We will see